spec/unit/filters/humanized_spec.rb

```diff
@@ -1,20 +1,46 @@
 require 'rails_helper'
 
 describe ActiveAdmin::Filters::Humanized do
-  let(:param) { ['category_id_eq', '1'] }
-
-  subject { ActiveAdmin::Filters::Humanized.new(param) }
+  let(:category) { Category.create!(name: 'Jazz') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.